deployer/hub.py

```diff
@@ -439,6 +439,13 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
         """
         Deploy this hub
         """
+        # Ensure helm charts are up to date
+        os.chdir("hub-templates/basehub")
+        subprocess.check_call(["helm", "dep", "up"])
+        if self.spec["template"] == "daskhub":
+            os.chdir("../daskhub")
+            subprocess.check_call(["helm", "dep", "up"])
+        os.chdir("../..")
```

Comment:
What about going to hub-templates, `helm dep up basehub` and `helm dep up daskhub` if the template is daskhub?
That should reduce the need to line 446 and mimic better what you are removing from the CI (and what we have with `support` above.
Just a suggestion :wink:

Reply:
Done!